src/lib/AppRegistry.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { AppRegistry, View, YellowBox } from "react-native"
 import { Container as RelayContainer } from "react-relay"
 
+import { SafeAreaInsets } from "lib/types/SafeAreaInsets"
```

Comment:
Curious why this was added?

Reply:
It's a Prop that's being passed to the Artwork view: https://github.com/artsy/emission/blob/master/src/lib/Scenes/Artwork/Artwork.tsx#L289